2 - RTB/Documentazione esterna/Analisi dei Requisiti/Analisi dei Requisiti.typ

```diff
@@ -158,7 +131,9 @@ Il prodotto sarà caratterizzato da:
 - *prodotti*:
   - rappresentano i reali prodotti contenuti nel magazzino;
   - contengono diverse informazioni riguardo il prodotto;
-  - sono contenuti in un bin e possono essere spostati verso un bin differente.
+  - sono contenuti in un bin e possono essere spostati verso un bin differente;
+  - è possibile ricercali tramite l'apposito pannello;
```

Comment:
Questi due punti li compatterei in uno unico. "- è possibile la ricerca dei prodotti parametrizzata: [inerire elenco dei parametri]"
I parametri individuati erano per id, per nome, per scaffale,